python/ray/ml/result.py

```diff
@@ -6,8 +6,9 @@
 class Result:
     """Result interface."""
 
-    checkpoint: Optional[Checkpoint]
-    metrics: Any
+    def __init__(self, metrics: Any, checkpoint: Optional[Checkpoint]):
```

Comment:
Add PublicAPI("beta") annotation.

Reply:
This is still alpha though.

Reply:
To clarify, my understanding, which is what we had done previously for Ray Train is that we don't add any annotations for alpha, and only added them once do beta release.

Reply:
We have this question as well. Can we clarify?

Reply:
@richardliaw I think we should put in beta annotations right off. Otherwise, the interface visibility is difficult to understand.

Reply:
Let's surface this higher at the slack channel.